packages/docs/components/fpo.svg

```diff
@@ -0,0 +1,5 @@
+<svg>
+  <rect width="100%" height="100%" style="fill: #fff; stroke:#a0a0a0;"></rect>
```

Comment:
We should prolly move these colors to a class on the `svg`

Reply:
this should never be visible, its just a placeholder deal - will be deleted before we release.